pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -389,7 +389,7 @@ func (p *PriorityQueue) flushUnschedulableQLeftover() {
 	}
 
 	if len(podsToMove) > 0 {
-		p.movePodsToActiveOrBackoffQueue(podsToMove, UnschedulableTimeout)
+		p.movePodsToActiveOrBackoffQueue(podsToMove, framework.WildCardEvent)
```

Comment:
@ahg-g I wonder how we differentiate the string representation of  `UnschedulableTimeout` and `WildCardEvent` event since the semantics is identical and String() method of `ClusterEvent` is based on this semantics

```
// String returns a string represerntation of ClusterEvent object
func (ce ClusterEvent) String() string {
	if ce.Resource == "" {
		return "InvalidEvent"
	}
	return fmt.Sprintf("%s:%s", ce.Resource, ce.ActionType.String())
}
```

Reply:
@minbaev is right. Unless we change the stringify implementation (for now it's implemented as `Resource:ActionType`, we will still see new labels showing in the metrics.

So not quite sure it's worth making the string representation (label of metrics) backwards compatible.

Reply:
Sorry, I meant a new resource type with the same semantics as `WildCard`. But that will be error prone.

> So not quite sure it's worth making the string representation (label of metrics) backwards compatible.

It is more about the readability of the label. All:* isn't useful. 

Here is another suggestion: 

Change `ClusterEvent` to include a `Label` field, and lets define constants for all events we are using and was listed in pkg/scheduler/internal/queue/events.go:

```
var PodAdd = ClusterEvent{Resource: Pod, ActionType: Add, Label: "PodAdd"}
...

var UnschedulableTimeout = ClusterEvent{Resource: WildCard, ActionType: All, Label: "UnschedulableTimeout"}
```

Reply:
Adding an explicit label looks good to me. And we need to add a helper function called `IsWildcardEvent()` to compare two events on their {Resource, ActionType}, to replace current occurrences of "if event == framework.WildCardEvent":

```
pkg/scheduler/internal/queue/scheduling_queue.go
868:	if clusterEvent == framework.WildCardEvent {
879:		evtMatch := evt == framework.WildCardEvent ||
```